rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```diff
@@ -660,7 +660,7 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 #
 # Regex notes: https://regex101.com/r/V6wrCO/1
 #
-SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx (?:[*?`\\'][^/\n]+/|\$[({\[#@!?*\-_$a-zA-Z0-9]|/[^/]+?[*?`\\'])" \
+SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx (?:[*?`\\'][^/\n]+/|\$[({\[#@!?*\-_$a-zA-Z0-9]|/[^/]+?[*?`[\\\\]'])" \
```

Comment:
As @RedXanadu pointed out, this looks weird. I thought I had understood the strategy but this looks wrong to me. Shouldn't it be `[*?`\\\\']`?

Reply:
I agree with @theseion. I think it _was_ ``[*?`\\']`` before the rebase, although there was also a merge conflict, so maybe that's where the alteration came from

Reply:
@RedXanadu @theseion Definitely, looks like that one slipped. Rebased and pushed again.